backend/config.go

```diff
@@ -151,6 +156,74 @@ func (c *GrafanaCfg) ConcurrentQueryCount() (int, error) {
 	return i, nil
 }
 
+type SqlConfig struct {
+	RowLimit                      int64
+	DefaultMaxOpenConns           int
+	DefaultMaxIdleConns           int
+	DefaultMaxConnLifetimeSeconds int
+}
+
+func (c *GrafanaCfg) Sql() (SqlConfig, error) {
+	// max open connections
+	maxOpenString, ok := c.config[SqlMaxOpenConnsDefault]
+	if !ok {
+		return SqlConfig{}, fmt.Errorf("SqlDatasourceMaxOpenConnsDefault not set in config")
+	}
+
+	maxOpen, err := strconv.Atoi(maxOpenString)
+	if err != nil {
+		return SqlConfig{}, fmt.Errorf("SqlDatasourceMaxOpenConnsDefault config value is not a valid integer")
+	}
+
+	// max idle connections
+	maxIdleString, ok := c.config[SqlMaxIdleConnsDefault]
+	if !ok {
+		return SqlConfig{}, fmt.Errorf("SqlDatasourceMaxIdleConnsDefault not set in config")
+	}
+
+	maxIdle, err := strconv.Atoi(maxIdleString)
+	if err != nil {
+		return SqlConfig{}, fmt.Errorf("SqlDatasourceMaxIdleConnsDefault config value is not a valid integer")
+	}
+
+	// max connection lifetime
+	maxLifeString, ok := c.config[SqlMaxConnLifetimeSecondsDefault]
+	if !ok {
+		return SqlConfig{}, fmt.Errorf("SqlDatasourceMaxConnLifetimeDefault not set in config")
+	}
+
+	maxLife, err := strconv.Atoi(maxLifeString)
+	if err != nil {
+		return SqlConfig{}, fmt.Errorf("SqlDatasourceMaxConnLifetimeDefault config value is not a valid integer")
+	}
+
+	rowLimitString, ok := c.config[SqlRowLimit]
+	if !ok {
+		return SqlConfig{}, fmt.Errorf("RowLimit not set in config")
```

Comment:
Nit: 
```suggestion
		return SqlConfig{}, errors.New("RowLimit not set in config")
```

That's more appropriate here. I know the other existing ones are doing that too, but they should also be updated. But is a idea to start that now 😄

Reply:
sure, fixed 👍